baybe/recommenders/pure/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""Pure recommenders."""
```

Comment:
As mentioned in the other file, one sentence about what a `pure` recommender is might be really valuable here.

Reply:
should I add it here or in `base.py` ?